package.json

```diff
@@ -30,5 +31,17 @@
   "devDependencies": {
     "@babel/cli": "^7.1.2",
     "@babel/preset-react": "^7.0.0"
-  }
+  },
+  "directories": {
+    "doc": "docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nourimeals/website.git"
+  },
+  "bugs": {
+    "url": "https://github.com/nourimeals/website/issues"
+  },
+  "homepage": "https://github.com/nourimeals/website#readme"
```

Comment:
Is homepage supposed to direct to our read me or is it meant to direct people to the deployed version?

Reply:
Do you have a preference? This wasn't something I setup but will update on request.

Reply:
@dawnaldrich We can leave it as-is for now.

Reply:
We can leave for now.